test/main_test.go

```diff
@@ -183,6 +182,10 @@ func showScenarios(features []*gherkin.Feature) {
 			mainLogger.Infof("Feature: %s", ft.Name)
 			for _, scenario := range ft.ScenarioDefinitions {
 				mainLogger.Info(framework.GetScenarioName(scenario))
+
+				if scenarioOutline, ok := scenario.(*gherkin.ScenarioOutline); ok {
+					mainLogger.Info(strings.Join(framework.GetExamplesNames(scenarioOutline), ", "))
```

Comment:
Wouldn't it have more sense to put the examples information into same line as the scenario? For example into the brackets? When they are placed on new line it may be harder to recognize that it is not a new scenario but rather example related to previous scenario.

Reply:
I was considering it, but I thought that for some reason only feature file has the prefix `Feature` specified and others are left without any further information, so I decided to leave it that way.
Then I would also add the prefix `Scenario` on the same line as the current output is something like:

```
"------------------ SHOW SCENARIOS ------------------"}
{"Feature: Kogito Service Performance"}
{"Spring Boot Kogito Service Performance without persistence"}
{"Unnamed Examples"}
{"Spring Boot Kogito Service Performance with persistence"}
{"Unnamed Examples"}
```

As you can see, even now you cannot tell that the second one is a Scenario as only the feature file has the prefix. I would maybe go for prefixing all of that? On the other hand you may know that if it is not prefixed with the `Feature`, it's Scenario or Scenario Outline :)

Or I can do something like: `Spring Boot Kogito Service Performance without persistence (Examples: <Examples...>)`

Reply:
`Spring Boot Kogito Service Performance without persistence (Examples: <Examples...>)` would be nice to have, IMHO more readable than current implementation.

Reply:
done :wink: